testing/python/fixtures.py

```diff
@@ -3950,3 +3950,27 @@ def fix():
 
     with pytest.raises(pytest.fail.Exception):
         assert fix() == 1
+
+
+# https://github.com/pytest-dev/pytest/issues/5036
+def test_fixture_param_shadowing(testdir):
```

Comment:
I suggest to put a description of the bug in the docstring, along with the issue number, something like:

```
Parametrized arguments would be shadowed if a fixture with the same name also exists (#5036).
```